### PR TITLE
Documents InternalsVisibleTo item

### PR DESCRIPTION
**Pull request by kzu**
Adds documentation for https://github.com/dotnet/sdk/pull/3439.

(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@kzu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/msbuild/common-msbuild-project-items.md [hunk 1]
```diff
@@ -125,6 +125,17 @@ In [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.m
...
+### InternalsVisibleTo
+ Specifies assemblies to be emitted as `[InternalsVisibleTo(..)]` assembly attributes.
+
+| Item metadata name | Description |
+|-----------------------| - |
+| Include | The assembly name. |
+| Key | Optional string. The public key of the assembly. |
+
+> [!NOTE] 
+> This applies to SDK-style projects only.
```
Comment (rainersigwald) [inline docs/msbuild/common-msbuild-project-items.md]: Is this the right phrasing? It applies to projects using the .NET Core SDK only, which is slightly distinct from SDK-style projects, but maybe that's too fine a distinction to draw?

**Comment by Mikejo5000** [comment]
@kzu Would you commit the suggestion? Thanks!

**Comment by Mikejo5000** [comment]
@kzu Daniel, would you commit the suggestion? Then we can merge. Thanks!

**Comment by Mikejo5000** [comment]
@ghogen Gordon, would you mind committing the suggestion? I don't have permissions for that. Thanks!

**Comment by jborsecnik** [comment]
@kzu, can you resolved the conflict?

**Comment by PRMerger6** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by bitbonk** [comment]
Could it me [this merge](https://github.com/MicrosoftDocs/visualstudio-docs/pull/3937/commits/00da4f1b4c534f7f8289e024d17c5965e8fb39b3) by @Mikejo5000 accidentally removed the documentation for the `InternalsVisibleTo` item in in csproj files. I can't find the documentation for this anywhere.

**Comment by kzu** [comment]
Indeed @bitbonk , the changes tab here show no changes at all: https://github.com/MicrosoftDocs/visualstudio-docs/pull/3937/files 😕

**Comment by ktoliver** [comment]
@Mikejo5000 It looks like resolving a merge conflict in this pull request from last year might have washed out the documentation the PR was meant to add. Could you take a look and suggest a path for fixing it, if that's what's needed? Thanks!

**Comment by Mikejo5000** [comment]
Hmm, yes, it looks like that's what might have happened. Added it back in MicrosoftDocs/visualstudio-docs-pr#7372

**Pull request merged**